assets/scss/components/_calendar.scss

```diff
@@ -335,20 +335,21 @@ $calendarWidth: 310px;
 .flatpickr-day:focus,
 .flatpickr-day.prevMonthDay:focus,
 .flatpickr-day.nextMonthDay:focus {
-	background: #e6e6e6;
-	border-color: #e6e6e6;
+	background: #707070;
+	border-color: #707070;
```

Comment:
for secondary color we can use `$C_secondary` constant, I think its will be better